README.md

```diff
@@ -57,3 +86,7 @@ Run tests with:
 ```sh
 rspec
```

Comment:
We are running tests on Travis with `bundle exec rake`. Does this make any difference? 

(Please, beware of the noob).

Reply:
Rake contents the rails tasks. The default task is to run the test suite, that in our case is the `rspec`. When we use `bundle exec rake` we're actually running `bundle exec rspec`. So in that case, whatever command you choose.

Reply:
Oh okay =) Thx <3